src/components.rs

```diff
@@ -0,0 +1,11 @@
+use specs::prelude::*;
+
+#[derive(Debug)]
+pub struct Transform {
+    pub x: f32,
+    pub y: f32,
+}
+
+impl Component for Transform {
+    type Storage = VecStorage<Self>;
```

Comment:
We could use [specs-derive](https://docs.rs/specs/0.15.0/specs/#the-data) to specify this. Just a little less verbose.